web/src/components/ZoneGauges.tsx

```diff
@@ -49,3 +50,10 @@ export default function ZoneGaugesWithCO2Square({
     </div>
   );
 }
+
+export default memo(
+  ZoneGaugesWithCO2Square,
+  (prevProps, nextProps) =>
+    JSON.stringify(prevProps.zoneData) === JSON.stringify(nextProps.zoneData) &&
+    prevProps.withTooltips === nextProps.withTooltips
+);
```

Comment:
The question here is if this costs more than it's worth. I'd be happy to delete it if that is the case.

Reply:
Decided to remove it and move the empty object creation outside the function so it's stable.